php/8.4.Dockerfile

```diff
@@ -58,6 +58,7 @@ RUN apt-get update && \
         webp \
         zip \
         php${PHP_VERSION} \
+        php${PHP_VERSION}-amqp \
```

Comment:
Bekommen wir das auch noch in das andere docker file

Reply:
Welches andere? In Dockerfile und 8.2.Dockerfile ist es bereits vorhanden

Reply:
https://github.com/netlogix/docker/blob/main/php/Dockerfile
https://github.com/netlogix/docker/blob/main/php/8.2.Dockerfile

Reply:
@saschanowak dort ist amqp bereits enthalten:
https://github.com/netlogix/docker/blob/main/php/Dockerfile#L57
https://github.com/netlogix/docker/blob/main/php/8.2.Dockerfile#L61

Reply:
Ahh ok sorry hatte ich nicht gesehen

Reply:
Builds sind grün